youtube_dl/extractor/rai.py

```diff
@@ -145,14 +145,30 @@ class RaiPlayIE(RaiBaseIE):
     }, {
         'url': 'http://www.raiplay.it/video/2016/11/gazebotraindesi-efebe701-969c-4593-92f3-285f0d1ce750.html?',
         'only_matching': True,
+    }, {
+        # DRM protected
+        'url': 'https://www.raiplay.it/video/2020/09/Lo-straordinario-mondo-di-Zoey-S1E1-Lo-straordinario-potere-di-Zoey-ed493918-1d32-44b7-8454-862e473d00ff.html',
+        'only_matching': True,
     }]
 
+    def _check_drm(self, obj):
+        if try_get(obj, lambda x: x['rights_management']['rights']['drm'], dict):
+            return True
+
+        if obj.get('program_info'):
+            return self._check_drm(obj['program_info'])
+
+        return False
```

Comment:
Inline.

Reply:
done in 614c48f3f497cb9537319057fadb28623d5f9d87